src/textmemes.ts

```diff
@@ -0,0 +1,58 @@
+import { CommandInteraction } from 'discord.js';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import Jimp from 'jimp';
+import Fs from 'fs';
+
+enum ReactionImage {
+  Rdj = 'rdj',
+}
+
+async function reactTextImage(
+  message: CommandInteraction,
+  path: string,
+  text: string,
+  xPos: number,
+  yPos: number,
+  maxWidth: number,
+) {
+  Jimp.read(path)
+    .then(async (image) => {
+      const loadedImage = image;
+      const font = await Jimp.loadFont(Jimp.FONT_SANS_16_BLACK);
+      loadedImage.print(font, xPos, yPos, text, maxWidth)
+        .write(`temp/${message.id}.png`);
```

Comment:
Let's use the system `/tmp` dir if possible. Should technically be some OS agnostic directory but in practice the bot is probably gonna run in a Linux environment.